client/scss/components/_transporter.scss

```diff
@@ -0,0 +1,68 @@
+.transporter {
+  background: color('white');
+  border-radius: $border-radius-unit;
+  padding: $vertical-space-unit map-get($gutter-width, 'small');
+  margin-bottom: $vertical-space-unit;
+
+  @include respond-to('medium') {
+    padding: $vertical-space-unit map-get($gutter-width, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding: $vertical-space-unit map-get($gutter-width, 'large');
+  }
+}
+
+.transporter__item:first-child:last-child {
```

Comment:
Yep, was just a way of applying a style if there was only one item (i.e. it's both the first and the last child)